### ISSUE TITLE
Define Input type through fieldTypes

### ISSUE THREAD

**Issue by ReptoxX** [seasonedcc/remix-forms#217]
Hello,

it's me again. Is there already a way to define a type for a field without having to customize the form entirely? 
I added another prop to my Form Function with `fieldTypes` so that i can specify `{{ password: 'password' }}` and having it rendered as a password input.

```ts
export function Form<Schema extends FormSchema>(
	props: FormProps<Schema> & { fieldTypes?: Partial<Record<keyof z.infer<Schema>, React.HTMLInputTypeAttribute>> }
) {
	const { t } = useTranslation();
	return (
		<RemixForm<Schema>
			fieldComponent={FormItemWrapper}
			labelComponent={Noop}
			inputComponent={FormInput}
			buttonComponent={FormButton}
			fieldErrorsComponent={Noop}
			globalErrorsComponent={FormError}
			buttonLabel={t('actions.save')}
			pendingButtonLabel={t('actions.saving')}
			{...props}
		>
			{({ Field, Button, Errors }) => {
				const fields = Object.keys(objectFromSchema(props.schema)).map((fieldName) => {
					return <Field name={fieldName} type={props.fieldTypes?.[fieldName] || 'text'} />;
				});
				return (
					<>
						{fields}
						<Button />
						<Errors className="mt-4" />
					</>
				);
			}}
		</RemixForm>
	);
}
```

Regards

**Comment by danielweinmann** [comment]
Hey, @ReptoxX! We don't have a way of doing that without adding children to the form or using renderField yet. And since renderField does not have access to the form's props, you wouldn't be able to accomplish exactly what you're trying to do.

I think adding a `fieldTypes` prop to `Form` is good. Are you interested in contributing a PR?

**Comment by ReptoxX** [comment]
Thanks for the fast answer @danielweinmann. I'm on it.

**Comment by ReptoxX** [comment]
Hey, i'm just wanna ask when this is going to be merged? Is there a roadmap or is something wrong with my pr?

**Comment by danielweinmann** [comment]
Hey, @ReptoxX! Sorry, I dropped the ball on this one. I'll review your PR later this week. Thank you for the reminder 🙏🏽

**Comment by danielweinmann** [comment]
Released in [2.2.0](https://github.com/seasonedcc/remix-forms/releases/tag/v2.2.0) thanks to @ReptoxX 🎉
